### PR TITLE
Revert "Revert "Upgrade to fused-effects-1.0.""

### PR DESCRIPTION
**Pull request by patrickt**
We need this for semantic’s FE-1.0 work, though don’t merge this just yet.

Reverts tree-sitter/haskell-tree-sitter#244.

### CODE REVIEW

File: .ghci [hunk 1]
```diff
@@ -5,4 +5,3 @@
 import TreeSitter.Unmarshal
 import Source.Span
 import Source.Range
-:set prompt "> "
```
Comment (robrix) [inline .ghci]: We should each set this in our per-user `.ghci` file rather than in any project-specific one.

File: cabal.project [hunk 2]
```diff
@@ -1,2 +1 @@
 packages: tree-sitter tree-sitter-*
-jobs: $ncpus
```
Comment (robrix) [inline cabal.project]: This is the default anyway.

**Review by robrix — COMMENTED** [review]
This is now ready for review ✨

**Review by aymannadeem — APPROVED** [review]
revert ( revert ( revert ( revert ( revert ( revert ( revert ...))))))

**Review by tclem — APPROVED** [review]

**Pull request merged**